test/integration/newapp_test.go

```diff
@@ -1711,7 +1712,7 @@ func TestNewAppSourceAuthRequired(t *testing.T) {
 	}
 }
 
-func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) string {
+func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) (string, func()) {
 	// Create test directories
 	testDir, err := ioutil.TempDir("", "gitauth")
```

Comment:
Thanks, I'll take a look.

Do you agree that temp files can be removed from these tests? Because just changing the location doesn't delete them but only makes it easy to remove them manually later.